jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js

```diff
@@ -209,19 +209,26 @@ YUI.add('user-profile-model-list', function() {
           </div>
         );
       }
-      const list = this.state.modelList;
-      if (!list || list.length === 0) {
-        return null;
-      }
-      const rows = list.map(this._generateRow);
       const props = this.props;
+      const acl = props.acl;
       let createNewButton;
-      if (props.canCreateNew) {
+      if (acl && acl.canAddModels()) {
```

Comment:
I'm not entirely sure if canAddModels will return false if readonly is true so to be on the safe side could you expand this to be:

`if (acl && !acl.isReadOnly() && acl.canAddModels()) {`